### PR TITLE
Release 0.2 update

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Update version in install instructions and manifests.

#### Which issue(s) this PR fixes:

Part of #352

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -164,7 +164,7 @@ ifndef ignore-not-found
   ignore-not-found = false
 endif
 
-clean-manifests = (cd config/components/manager && $(KUSTOMIZE) edit set image controller=gcr.io/k8s-staging-kueue/kueue:main)
+clean-manifests = (cd config/components/manager && $(KUSTOMIZE) edit set image controller=gcr.io/k8s-staging-kueue/kueue:release-0.2)
```
Comment (ArangoGutierrez) [inline Makefile:167]: Why moving from main here?

File: config/components/manager/kustomization.yaml [hunk 2]
```diff
@@ -14,4 +14,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: gcr.io/k8s-staging-kueue/kueue
-  newTag: main
```
Comment (ArangoGutierrez) [inline config/components/manager/kustomization.yaml:17]: same question

**Review by ArangoGutierrez — COMMENTED** [review]
just 2 nit questions

File: Makefile [hunk 3]
```diff
@@ -164,7 +164,7 @@ ifndef ignore-not-found
   ignore-not-found = false
 endif
 
-clean-manifests = (cd config/components/manager && $(KUSTOMIZE) edit set image controller=gcr.io/k8s-staging-kueue/kueue:main)
```
Comment (ahg-g) [inline Makefile:167]: why was this set to main and not 0.1.1?

Comment (ahg-g) [reply Makefile:167]: ditto the other instances below

File: Makefile (hunk 1 above)
Comment (alculquicondor) [reply Makefile:167]: Oops wrong target branch. This is for the release-0.2 branch.

File: Makefile (hunk 3 above)
Comment (alculquicondor) [reply Makefile:167]: This is just for the release-0.2 branch (I fixed the target).

File: config/components/manager/kustomization.yaml (hunk 2 above)
Comment (alculquicondor) [reply config/components/manager/kustomization.yaml:17]: same answer :sweat_smile:

File: Makefile (hunk 1 above)
Comment (ArangoGutierrez) [reply Makefile:167]: aaaahhhh ;P

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**